api.md

```diff
@@ -1,22 +1,524 @@
 # OWAPI API Docs
 
-##V2
+## Notes
 
-** All times are in *HOURS* unless specified otherwise. **
-
-#### Note about Regions & Platforms
+Regions and platforms can be overridden with URL parameters.
 
-Regions and platforms can be overriden with URL parameters.
+#### Regions *(V2)
 
-For example, if you wanted to force fetching of US stats:
+For example, if you wanted to force fetching of US stats, add `?region=us`:
 
 ```
 $ http GET "https://owapi.net/api/v2/u/SunDwarf-21353/stats/general?region=us"
 ```
 
-Note that if they are not overriden, the API will automatically determine which region to use.
+Note that if they are not overridden, the API will automatically determine which region to use.
+
+#### Regions *(V3)
+
+In V3, all regions are automatically checked. The region that user is in returns a JSON array, the other regions return `null`. See V3 results below.
```

Comment:
V3 returns data for all regions the user is in, not just one, so my profile has US and EU data, and the KR field is None, rather than the other two being null. Should probably clarify this.